lib/node_modules/@stdlib/blas/ext/base/dsumkbn/benchmark/benchmark.js

```diff
@@ -44,7 +45,7 @@ var rand = uniform( -10.0, 10.0 );
 * @returns {Function} benchmark function
 */
 function createBenchmark( len ) {
-	var x = filledarrayBy( len, 'float64', rand );
+	var x = uniform( len, -100, 100, options );
```

Comment:
Wouldn't the range be from `-10` to `10`?

Reply:
I see you're coming from the previous `rand` function. IIRC I've been using -100 to 100 in every similar package `README`, `examples` and `Benchmarks`. So in terms of consistency we're good.

Reply:
Yeah, for this package, fine to adjust the range to -100 to 100.